mptt/admin.py

```diff
@@ -84,3 +89,174 @@ def get_actions(self, request):
                 'delete_selected',
                 _('Delete selected %(verbose_name_plural)s'))
         return actions
+
+
+class DraggableMPTTAdmin(MPTTModelAdmin):
+    """
+    The ``DraggableMPTTAdmin`` modifies the standard Django administration
+    change list to a drag-drop enabled interface.
+    """
+
+    list_per_page = 2000  # This will take a really long time to load.
+
+    def __init__(self, *args, **kwargs):
+        super(DraggableMPTTAdmin, self).__init__(*args, **kwargs)
+
+        opts = self.model._meta
+        self.change_list_template = [
+            'admin/%s/%s/draggable_mptt_change_list.html' % (
+                opts.app_label, opts.object_name.lower()),
+            'admin/%s/draggable_mptt_change_list.html' % opts.app_label,
+            'admin/draggable_mptt_change_list.html',
+        ]
+
+        # Set a better header than "title"
```

Comment:
So this sets the header of the first column to the name of the model?

It seems wrong to be changing `self.__class__...` in an `__init__` method. There must be a way to decorate the instance-bound method instead of the unbound function?

I think it should use the `short_description` of the first column name from `list_display`, rather than the name of the model. Then subclasses can override it easily.